test/specs/test.js

```diff
@@ -24,6 +24,13 @@ module.exports = {
       // should expand root by default
       .assert.count('.instance', baseInstanceCount)
 
+      // prop types
+      .click('.instance .instance:nth-child(2) .self')
+      .assert.containsText('.component-name', 'Target')
+      .assert.containsText('.data-field:nth-child(5)', 'msg:"hi"')
+      .assert.containsText('.data-field:nth-child(6)', 'obj:undefined')
+      .assert.containsText('.data-field:nth-child(2)', 'ins:Object')
```

Comment:
I tried to test the type content with `.assert.containsText('.data-field:nth-child(2) .self .prop .meta-field:nth-child(1) .value', 'MyClass')` but the content is always empty although it does appear in the Devtools as `MyClass`